submissions/oluwaboriShobayo/React/to-do-app/src/Components/Form.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+const Form = ({setInputText, todos, setTodos,  inputText }) => {
+
+// Function that updates the input field
+    const inputFieldHandler =(e) => {
+        console.log(e.target.value);
+        setInputText(e.target.value);
+    };
+ const submitHandler = (e) => {
+        e.preventDefault();
+        setTodos ([
+                ...todos, {text:inputText, completed:false, id:Math.random()}
```

Comment:
I think you should extend this Math.random to like 10,000 so that the likelihood of having a duplicate index is slim